### PR TITLE
Fix timm_nfnet train batch size and improve the entire model quality

### PR DESCRIPTION
**Pull request by xuzhao9**
# Train

batch size: 128 
Source
- https://github.com/rwightman/pytorch-image-models/blob/master/train.py#L114
- https://gist.github.com/rwightman/bb59f9e245162cee0e38bd66bd8cd77f#file-bench_by_train-csv-L147

## Latency
GPU Time:            2767.694 milliseconds
CPU Dispatch Time:   2767.542 milliseconds
CPU Total Wall Time: 2767.665 milliseconds

## Profile
image

# Eval
batch size: 256 
- source: https://github.com/rwightman/pytorch-image-models/blob/master/results/model_benchmark_amp_nchw_rtx3090.csv

## Latency
GPU Time:            2209.692 milliseconds
CPU Dispatch Time:    15.806 milliseconds
CPU Total Wall Time: 2209.685 milliseconds

## Profile
image

### CODE REVIEW

**Comment by xuzhao9** [comment]
Full traces: 
[timm_nfnet.tar.gz](https://github.com/pytorch/benchmark/files/7603390/timm_nfnet.tar.gz)

File: torchbenchmark/models/timm_nfnet/__init__.py [hunk 1]
```diff
@@ -1,87 +1,115 @@
...
+        raise NotImplementedError("Disable the train test because it causes CUDA OOM on Nvidia T4")
+        self.eval_model.eval()
+        with torch.no_grad():
+            for _, (input, _) in zip(range(self.args.eval_num_batch), self.loader_eval):
+                return self.eval_model, (input, )
 
+    # Temporarily disable training because this will cause CUDA OOM in CI
+    # TODO: re-enable this test when better hardware is available
     def train(self, niter=1):
+        raise NotImplementedError("Disable the train test because it causes CUDA OOM on Nvidia T4")
```
Comment (aaronenyeshi) [inline torchbenchmark/models/timm_nfnet/__init__.py:88]: Would this also disable the test for CPU?

File: torchbenchmark/models/timm_nfnet/__init__.py [hunk 2]
```diff
@@ -1,87 +1,115 @@
...
+        
+        # Disable train data load as there is not enough GPU memory
+        # TODO: enable with larger GPU
+        # self.loader_train = prefetch_loader(self.loader_train, device)
+        # self.loader_validate = prefetch_loader(self.loader_validate, device)
+        self.loader_eval = prefetch_loader(self.loader_eval, device)
 
     def get_module(self):
-        return self.model, (self.cfg.example_inputs,)
+        raise NotImplementedError("Disable the train test because it causes CUDA OOM on Nvidia T4")
```
Comment (aaronenyeshi) [inline torchbenchmark/models/timm_nfnet/__init__.py:79]: same as other comment, how about CPU here?

**Review by aaronenyeshi — APPROVED** [review]
LGTM! Small question, but won't block you on it.

File: torchbenchmark/models/timm_nfnet/__init__.py (hunk 1 above)
Comment (xuzhao9) [reply torchbenchmark/models/timm_nfnet/__init__.py]: Thanks for the comment! I have enabled it for CPU.

File: torchbenchmark/models/timm_nfnet/__init__.py (hunk 2 above)
Comment (xuzhao9) [reply torchbenchmark/models/timm_nfnet/__init__.py]: same as above, cpu test is enabled

**Pull request closed without merging**